cosmoplots/colors.py

```diff
@@ -0,0 +1,84 @@
+import numpy as np
+from colour import Color
+from matplotlib.colors import LinearSegmentedColormap
+import matplotlib.pyplot as plt
+from matplotlib import cm, colors
+from typing import List
+
+
+def make_color_swatch(color_list: list):
+    """
+    Generate plot of the desired color swatch
```

Comment:
Could you provide more details about when you would use this function and what is the benefit of it?

Reply:
I've added more detail now